_utility/normalize_bucket_name.sh

```diff
@@ -0,0 +1,6 @@
+normalizedBucketName() {
```

Comment:
Might not be necessary to encapsulate into a separate function, but I'll leave this for you to decide.